rock-n-rollup/src/core/application.rs

```diff
@@ -12,7 +12,7 @@ where
     base: Service<R, Vec<u8>, ()>,
 }
 
-impl<'a, R: Runtime + 'static> Application<'a, R> {
+impl<'a, R: Runtime> Application<'a, R> {
```

Comment:
Very nice, using static everywhere is not the best things to do in rust, thanks for the change